modules/expectations/session_expectations.lua

```diff
@@ -21,6 +21,9 @@ local mt = { __index = { } }
 --- Type which represents single mobile expectation
 -- @type MobileExpectations
 
+--- Expectation of frame event
+-- @tparam table frameMessage Frame message table to expect
```

Comment:
@aderiabin not all params listed

Reply:
@LuxoftAKutsan  Done (see ab761db127418d0f025c8e5768470f626587e41c)